ui/src/components/dynamics-form/FormItem.vue

```diff
@@ -127,7 +128,7 @@ const rules = computed(() => {
     ? props_info.value.rules.map(to_rule)
     : {
         message: errMsg.value,
-        trigger: 'blur',
+        trigger: ['blur', 'change'],
         required: props.formfield.required === false ? false : true
       }
 })
```

Comment:
The code is generally clean and concise, but there's a small issue with the `rules` computed property. If no rules are provided via `props_info.value.rules`, it currently returns an empty object instead of using the default values specified in the comments (`message: errMsg.value`). You can update this to use those defaults properly:

```javascript
const rules = computed(() => {
  if (!props_info || !props_info.value || !props_info.value.rules) {
    return {
      message: errMsg.value,
      // Use default triggers based on comment here.
      trigger: props_formfield.trigger ?? 'change',
      required: !!props.formfield.required
    };
  } else {
    return props_info.value.rules.map(to_rule);
  }
});
```

Additionally, you may want to log some debugging information if something goes wrong during the execution to help identify issues in further checks.

This change ensures that the function behaves as expected when no rules are provided, which might be important depending on your application logic.